### DISCUSSION TITLE
unknown flag: --remove-orphans

### DISCUSSION THREAD

**Discussion by reflexiveClick** [moyix/fauxpilot#170] [Q&A]
I just installed fauxpilot-main through WSL and when I try to do the following command:
`bash /mnt/d/KoboldAI/fauxpilot-main/launch.sh`
It gives me the following error and does not launch fauxpilot:
`unknown flag: --remove-orphans`

Any help would be greatly appreciated!

**Comment by mastercoder8** [comment]
This might happen if you have docker-compose v1, consider updating it to latest version or use [compose-switch](https://github.com/docker/compose-switch)
